chingu-voyages/v52-tier3-team-38
# Gary/feature/toggle views

# Description
This PR adds functionality to switch between Lists and Map views with a single button. This is built on the previous PR.

## Related to the following Jira/Trello issue 
https://trello.com/c/P9MUxllH/33-view-resident-requests-map-view

## Main changes explained:
- Removed admin/requests as it was no longer necessary.
- Created MapView and ListView routes where AppointmentMapView and AppointmentListView can be found.
- Created SwitchViewButton component to switch routes where the views are found.
- Adjusted page.js to redirect to "admin/appointments/listView when admin is logged in.

## How to test:
1. check into current branch
2. do `npm install` and `npm run dev` to run this PR locally
3. Login as admin, the list view should be shown and the url should be admin/appointments/listView.
4. Press the Switch view button on the bottom left corner, the page should redirect to admin/appointments/listView. Toggling again should send you back to the list view page.
